### DISCUSSION TITLE
Controls not updating properly when object is locked

### DISCUSSION THREAD

**Discussion by rwmorton** [fabricjs/fabric.js#8763] [Q&A]
Hi there, so I have these objects that I am locking but the controls don't seem to behave properly. It seems that when there is an empty selection and you click the object the controls work as expected. However, if you select two locked objects, one after the other you can modify the second object (unless you click it again). See attached screencapture.

I am locking the object like so:

![image](https://user-images.githubusercontent.com/95928533/223339563-168eca33-d8cf-428a-92e3-fab7568ff3fe.png)

[locking-issue.webm](https://user-images.githubusercontent.com/95928533/223339693-3338a680-d452-47ce-ac00-a0b0355b6c72.webm)

**Comment by rwmorton** [comment]
As another example - with hasControls = false you can see that the controls somehow don't update properly. Not sure how to get this working?

[has-controls-false.webm](https://user-images.githubusercontent.com/95928533/223348777-4ec62513-9ff3-4a5d-bd39-7148c2e0a45a.webm)

**Comment by ShaMan123** [comment]
Please provide a minimal reproduction in v6

**Reply by rwmorton** [reply]
Hi thanks for the response! I can't do this as the fabric version we are using is monkey patched so things are sort of a mess. I found what the problem was - it's this issue we have with selections not updating the bounding box and I had written some code to override it. We are planning to upgrade our fabric version but that's only some time down the line as it's going to be a lot of work to do this upgrade.
